src/evaluation/baselines.py

```diff
@@ -96,7 +96,7 @@ def solve(self, input: Input, **kwargs):
                 r1 = self.actions.wait_for_element(input)
 
                 # wait 0.1 sec for the page to fully load
-                sleep(0.1)
+                sleep(3)
```

Comment:
This may make things a lot slower though

Reply:
Nvm saw your slack message.

Reply:
No you're 100% right! This shouldn't have made it into prod, this is only for run_single when analyzing a single instance. Fixing right now